src/openms/source/CHEMISTRY/ElementDB.cpp

```diff
@@ -279,21 +281,30 @@ namespace OpenMS
 
     // build last element
     double avg_weight(0), mono_weight(0);
-    IsotopeDistribution isotopes = parseIsotopeDistribution_(Z_to_abundancy);
+    IsotopeDistribution isotopes = parseIsotopeDistribution_(Z_to_abundancy,Z_to_mass);
```

Comment:
Can you make the variable name consistent? Even if it might not be your fault. Either Z_to_abundancy or Z_to_abundance.